src/Components/ClusterRules/ClusterRules.js

```diff
@@ -125,7 +127,11 @@ const ClusterRules = ({ cluster }) => {
       .map((value, key) => [
         {
           rule: value,
-          isOpen: isAllExpanded,
+          isOpen: isAllExpanded
+            ? true
+            : value?.rule_id === recordExpanded.includes(value?.rule_id)
+            ? true
+            : false,
```

Comment:
down with ternary overuse 
```suggestion
          isOpen: isAllExpanded || value?.rule_id === recordExpanded.includes(value?.rule_id);
```